### DISCUSSION TITLE
Sound actuator on sport off  the other modes

### DISCUSSION THREAD

**Discussion by lito69** [jilleb/MQB-FPA#54] [Modified profiles]
Hello, I have placed a soundactuator in my seat leon 2018, it works, my gw dataset has to ask the soundactuator for comfort, normal, sport and eco, I have modified it so that in individual only choose sport and comfort, modes in eco and normal it doesn't sound, in sport yes, I would like to change the text on the screen so that it says activated and deactivated and that it only sounds in sport, the modes that I don't want it to sound I have put config8 (copied passat dataset), in sport config9, with this I get it to say activated and deactivated on the screen but it doesn't work, in hmi shows I have put the equivalent in each mode to config8 and 9, in individual controls research at the bottom I have put 0500 and 0a00 to choose between sport and comfort, I don't know if I am doing it right, in request values ​​8fb the passsat has interior sound 0.0.0.0.152 for on and off, I have tried 0.0.0.0.50 and I get that it does not sound activated and it sounds deactivated, I have it the other way around and I also think it sounds normal, no sport, you know how to place it correctly entity? 8fb I don't really know how it works, I don't understand it. Thanks!

**Comment by olli991** [comment]
You need to change the request values to get this behavior.
05 is off for soundactor

**Reply by lito69** [reply]
> Debe cambiar los valores de solicitud para obtener este comportamiento. 05 está apagado para el actor de sonido

Hello, thanks for answering, I ended up finding out how it worked, they are only mode numbers grouped in two in each byte, normally 05 and 01 sent to the sak is without sound, now I have one that sends 2 and 3 and in the mib2 it shows on and off, I don't understand why the dataset has 08 and 09 but sends 2 and 3 to the sak, but it works.

**Reply by olli991** [reply]
It also depends on the soundactor firmwsre you're running. Some have more sounds and some less. You actually need to match this. 

Some sound actor firnware has a sound for 01 (for example) and some other don't. 

It's always a play of both ends (soundactor firnware & fpa request values)

**Reply by lito69** [reply]
> También depende de la firma de actor de sonido que estés ejecutando. Algunos tienen más sonidos y otros menos. En realidad necesitas hacer coincidir esto.
> 
> Algunos firmware de actor de sonido tienen un sonido para 01 (por ejemplo) y otros no.
> 
> Siempre es un juego de ambos extremos (firmware de soundactor y valores de solicitud de fpa)

Yes, I modify the gw dataset for on and off and in the sak I put sound only in sport (for those who have several sounds), but what I don't understand is why in the on off dataset that I have it puts 08 and 09 for on and off but really to the sak unit it sends number 2 and 3 as driver profile, which is in dataset gw that transforms 08 and 09 into 02 03.

**Reply by olli991** [reply]
Can you attach your current dataset so that I can have a look please

**Reply by lito69** [reply]
> Can you attach your current dataset so that I can have a look please

Yes, I am sending you the dataset that, despite being configured in 08 and 09, sends 02 and 03, the logical thing is to think that it sends 08 and 09 to the sak, but it sends 02 and 03, it is the only one that I have seen that in the mib it shows on and off but it actually sends 02 and 03, it's just what I want but I don't understand where it does the conversion from 08 to 02.

[Golf gti ac con amb y sak on off en grupo funciona sak direccion blanda hvac comfort.zip](https://github.com/jilleb/MQB-FPA/files/9544746/Golf.gti.ac.con.amb.y.sak.on.off.en.grupo.funciona.sak.direccion.blanda.hvac.comfort.zip)

**Reply by olli991** [reply]
Currently you engine sound is in a group. So you have two settings... the request values and the group "child" values. Both need to match up... that is currenlty not the case on your file.

The request values are sending 08 and 09.
But the group child request values are sending 02 and 03. 

That explains your findings :) Change one or the other and it will match up.
